<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PaymentMethodWindow</class>
 <widget class="QMainWindow" name="PaymentMethodWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PaymentW</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>7</y>
      <width>83</width>
      <height>66</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>40</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background:transparent;}
QPushButton{border-image: url(:/none.png);}
QPushButton:checked{border-image: url(/home/pi/Charger/JPG/PNG/Rect2.png);}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>160</y>
      <width>301</width>
      <height>251</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>40</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;
border-image: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>160</y>
      <width>301</width>
      <height>251</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>40</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;
border-image: transparent;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="chgrNumLabelSm">
    <property name="geometry">
     <rect>
      <x>45</x>
      <y>170</y>
      <width>171</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Piboto</family>
      <pointsize>48</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string notr="true">888</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="chgrNumLabelLg">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>170</y>
      <width>321</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>PibotoLt</family>
      <pointsize>72</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>000</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="qrLabelSm">
    <property name="geometry">
     <rect>
      <x>488</x>
      <y>138</y>
      <width>96</width>
      <height>96</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>QR Label</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="serNumLabelSm">
    <property name="geometry">
     <rect>
      <x>462</x>
      <y>246</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>PibotoLt</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Charger Serial No</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="qrLabelLg">
    <property name="geometry">
     <rect>
      <x>88</x>
      <y>154</y>
      <width>185</width>
      <height>185</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>QR Label</string>
    </property>
    <property name="scaledContents">
     <bool>false</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="serNumLabelLg">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>350</y>
      <width>225</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>PibotoLt</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Charger Serial No</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="connStatusIcon">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>410</y>
      <width>64</width>
      <height>64</height>
     </rect>
    </property>
    <property name="text">
     <string>Conn Status</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
